cnd/src/network.rs

```diff
@@ -618,8 +617,6 @@ impl libp2p::swarm::NetworkBehaviourEventProcess<orderbook::BehaviourOutEvent> f
                     }
                 });
 
-                self.confirmed_order_peers.insert(order_id, peer_id);
```

Comment:
Its a little hard to review this PR because there is more than one thing happening, I know there is a lot of clean up work being done in the orderbook so this is kind of expected however removal of this line is strange - can you please explain why it was removed?